### PR TITLE
Add calico support

### PR DESCRIPTION
**Pull request by mazdakn**
Add Calico support to the network policy framework. It count's the number of iptables in filter and raw table which are used for implementation for different k8s policies.

### CODE REVIEW

File: kube-burner-workload/calico/metrics.yml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+- query: histogram_quantile(0.99, sum(irate(apiserver_request_duration_seconds_bucket{apiserver="kube-apiserver", verb=~"LIST|GET", subresource!~"log|exec|portforward|attach|proxy"}[2m])) by (le, resource, verb, scope)) > 0
+  metricName: readOnlyAPICallsLatency
+
+- query: histogram_quantile(0.99, sum(irate(apiserver_request_duration_seconds_bucket{apiserver="kube-apiserver", verb=~"POST|PUT|DELETE|PATCH", subresource!~"log|exec|portforward|attach|proxy"}[2m])) by (le, resource, verb, scope)) > 0
+  metricName: mutatingAPICallsLatency
+
+- query: sum(irate(apiserver_request_total{apiserver="kube-apiserver",verb!="WATCH"}[2m])) by (verb,resource,code) > 0
+  metricName: APIRequestRate
+
+# Containers & pod metrics
```
Comment (jtaleric) [inline kube-burner-workload/calico/metrics.yml:15]: Should we update the metrics to also include Calico pod names?

File: kube-burner-workload/calico/metrics.yml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+- query: ovs_vswitchd_rconn_discarded
+  metricName: ovsVswitchdRconnDiscarded
+
+- query: ovs_vswitchd_rconn_overflow
+  metricName: ovsVswitchdRconnOverflow
+
+- query: ovs_vswitchd_stream_open
+  metricName: ovsVswitchdStreamOpen
+
```
Comment (jtaleric) [inline kube-burner-workload/calico/metrics.yml]: Should we add Calico specific prom queries?

**Comment by mazdakn** [comment]
@jtaleric thanks for the initial review. I addressed your comments, and also made more changes. I can now run the whole setup without any issue. Please take another look.

File: kube-burner-workload/calico/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+## Running
+
+1. This profile assumes you have an openshift cluster, and the KUBECONFIG that can be used in the scale test.
```
Comment (npinaeva) [inline kube-burner-workload/calico/README.md]: I guess you want to search for openshift in this folder and replace with calico :)

File: kube-burner-workload/calico/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+2. Build kube-burner from the current branch
+      `make build`
+3. `cd ./examples/workloads/network-policy`
+4. Set env variables with the test config in the `env` file
+
+   4.1 Set env file variable PLATFORM=calico
+
+5. Set env variables in the `calico/env` file
+6. `source ./env`
+7. This command uses `oc` binary which is an Openshift CLI similar to kubectl
```
Comment (npinaeva) [inline kube-burner-workload/calico/README.md]: this command can probably refer to the new kind-metrics folder or copy some data form there to explain where  prometheus ip and ES variables are coming from

**Review by npinaeva — COMMENTED** [review]
great to see the next platform!
If you want to include dashboards and metrics, you probably also need a local env file with ES_SERVER and ES_INDEX variables

File: kube-burner-workload/calico/README.md (hunk 3 above)
Comment (mazdakn) [reply kube-burner-workload/calico/README.md]: Thanks, Fixed it.

File: kube-burner-workload/calico/README.md (hunk 4 above)
Comment (mazdakn) [reply kube-burner-workload/calico/README.md]: Thanks. Fixed it. I'm going to update the README file when we have finalised the tests.

**Comment by npinaeva** [comment]
that lgtm, but I think we should merge https://github.com/npinaeva/k8s-netpol-scale/pull/5 and https://github.com/npinaeva/k8s-netpol-scale/pull/6 first, then rebase and test this one to ensure all timeouts and scripts work properly, wdyt?

**Pull request merged**